rs/tests/system_tests.bzl

```diff
@@ -33,26 +33,33 @@ def _run_system_test(ctx):
         content = """#!/bin/bash
             set -eEuo pipefail
 
-            # for every ic-os image specified, first ensure it's in remote storage, then
-            # export its download URL as an environment variable.
-            if [ -n "$ICOS_IMAGES" ]; then
-              # split the ";"-delimited list of "envvar:filepath;envvar2:filepath2;..."
+            # Resolve any RUN_SCRIPT_ variables
+
+            # RUN_SCRIPT_REPO_ICOS_IMAGES:
+            # For every repo image specified, first ensure it's in remote
```

Comment:
what's a repo image?

Reply:
I was trying to capture that these images are based in the current repo, versus mainnet images, or any other arbitrary URLs.

Maybe `repo_based_icos_images` is a better name? The variable was just getting a little long. `repo_based_imgs`?

Reply:
I'm not even sure we need to make that distinction, it's not like we have tens of different image types. Bit subjective but it seems to introduce a bit of complexity, i.e. forces me to think just a bit longer about what this means

Reply:
I've set this back to the simple naming